app/src/test/java/org/simple/clinic/illustration/HomescreenIllustrationRepositoryTest.kt

```diff
@@ -88,4 +88,16 @@ class HomescreenIllustrationRepositoryTest {
         .assertNoValues()
         .assertNoErrors()
   }
+
+  @Test
+  fun `verify illustration is not set when there is event today but no illustration file`() {
+    whenever(fileStorage.getFile(eventId)).thenReturn(GetFileResult.Failure(FileNotFoundException()))
```

Comment:
See earlier comment about the behaviour of the `FileStorage` class. Let's talk about this next week.